### PR TITLE
Ignore dsn.Connect for setting externalauthentication

### PR DESCRIPTION
**Pull request by MichaelS11**

### CODE REVIEW

**Comment by MichaelS11** [comment]
@mattn 

Please review and merge

File: oci8.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func ParseDSN(dsnString string) (dsn *DSN, err error) {
 		}
 	}
 
-	if len(dsn.Username)+len(dsn.Password)+len(dsn.Connect) == 0 {
+	if len(dsn.Username)+len(dsn.Password) < 1 {
```
Comment (mattn) [inline oci8.go]: Seems good to me but please keep == 0

**Comment by MichaelS11** [comment]
@mattn There you go

**Comment by mattn** [comment]
Thanks!

**Pull request merged**